### DISCUSSION TITLE
I want to enable claiming in the END

### DISCUSSION THREAD

**Discussion by noaskers** [GriefPrevention/GriefPrevention#2185] [General]
I want to enable claiming in the END

But i want to make it so the MAIN island can't be claimed but i still want to make it so that they can respawn the dragon & break end crystals & make it so they can go to the portals so any suggestions for this?

Already much thanks!

**Comment by QarthO** [comment]
Turn on admin claiming mode
Claim the entire island
Stand inside of it
Run the command /trust public
